bitnami/pgbouncer/1/debian-11/rootfs/opt/bitnami/scripts/libpgbouncer.sh

```diff
@@ -217,9 +217,13 @@ pgbouncer_initialize() {
             database_value+=" connect_query='${PGBOUNCER_CONNECT_QUERY}'"
         fi
         ini-file set --section "databases" --key "$PGBOUNCER_DATABASE" --value "$database_value" "$PGBOUNCER_CONF_FILE"
+        i=0; while true; VAR_NAME="DATABASE${i}_DSN"; do if [ -z "${!VAR_NAME+x}" ]; then break; else dsn=${!VAR_NAME}; echo \$dsn; ini-file set --section databases --key "$(echo $dsn | cut -d = -f 1)" --value "$(echo $dsn | cut -d = -f 2-)" "$PGBOUNCER_CONF_FILE"; i=$(( $i + 1 )); fi; done;
```

Comment:
Could you separate the code into different lines to make it clearer, please?

Reply:
Done.
But I haven't tested bash syntax correctness

Reply:
I will check it. Thank you.